Makefile.in

```diff
@@ -52,11 +52,19 @@ endif
 DCFLAGS += $(NOTIF_VERSIONS)
 endif
 
-system_unit_files = systemd.units/onedrive@.service
-user_unit_files = systemd.units/onedrive.service
+system_unit_files = systemd.units/onedrive.service
+user_unit_files = systemd.units/onedrive@.service
```

Comment:
No, this is wrong. These are the correct settings. WIth the `@` , root can start a onedrive process for a specific user
```
sudo systemctl enable onedrive@USER
```
and a normal user can do
```
systemctl --user enable onedrive
```

Turning this around is incorrect.

I am quite sure that RH/CentOS is **not** getting this wrong, neither.

Reply:
fixed